packages/webapp/src/pages/election/components.tsx

```diff
@@ -57,13 +57,17 @@ export const MembersPage = (props: Props) => {
                 )}
             </MembersGrid>
             <Container className="pt-6 border-t border-b">
-                <Heading size={1}>Winner Chip</Heading>
+                <Heading size={1}>Delegate Chip</Heading>
                 {members.isLoading && "Loading members..."}
                 {members.error && "Fail to load members"}
             </Container>
             <MembersGrid members={members.data?.slice(1, 2)}>
                 {(member) => (
-                    <WinningMemberChip key={member.account} member={member} />
+                    <DelegateChip
+                        key={member.account}
+                        member={member}
+                        level="D3 - Chief Delegate"
```

Comment:
Similar here w the hard coded value

Reply:
Yes. This is just example UI for now. Not hooked up to any data. We'll make sure these are dynamic and are a matter of configuration. 👍